packages/legend-extension-dsl-serializer/src/models/protocols/pure/DSLSerializer_PureProtocolProcessorPlugin.ts

```diff
@@ -0,0 +1,305 @@
+          return BINDING_ELEMENT_CLASSIFIER_PATH;
+        } else if (elementProtocol instanceof V1_SchemaSet) {
+          return SCHEMA_SET_ELEMENT_CLASSIFIER_PATH;
+        }
+        return undefined;
+      },
+    ];
+  }
+
+  override V1_getExtraElementProtocolSerializers(): V1_ElementProtocolSerializer[] {
+    return [
+      (
+        elementProtocol: V1_PackageableElement,
+      ): PlainObject<V1_PackageableElement> | undefined => {
+        if (elementProtocol instanceof V1_Binding) {
+          return serialize(V1_bindingModelSchema, elementProtocol);
+        } else if (elementProtocol instanceof V1_SchemaSet) {
+          return serialize(V1_schemaSetModelSchema, elementProtocol);
+        }
+        return undefined;
+      },
+    ];
+  }
+
+  override V1_getExtraElementProtocolDeserializers(): V1_ElementProtocolDeserializer[] {
+    return [
+      (
+        json: PlainObject<V1_PackageableElement>,
+      ): V1_PackageableElement | undefined => {
+        if (json._type === V1_BINDING_ELEMENT_PROTOCOL_TYPE) {
+          return deserialize(V1_bindingModelSchema, json);
+        } else if (json._type === V1_SCHEMA_SET_ELEMENT_PROTOCOL_TYPE) {
+          return deserialize(V1_schemaSetModelSchema, json);
+        }
+        return undefined;
+      },
+    ];
+  }
+
+  override V1_getExtraElementTransformers(): V1_ElementTransformer[] {
+    return [
+      (
+        metamodel: PackageableElement,
+        context: V1_GraphTransformerContext,
+      ): V1_PackageableElement | undefined => {
+        if (metamodel instanceof Binding) {
+          const protocol = new V1_Binding();
+          V1_initPackageableElement(protocol, metamodel);
+          protocol.name = metamodel.name;
+          protocol.package = metamodel.package?.fullPath ?? '';
+          protocol.schemaId = metamodel.schemaId;
+          protocol.schemaSet = metamodel.schemaSet;
+          protocol.contentType = metamodel.contentType;
+          protocol.modelUnit = metamodel.modelUnit;
+          return protocol;
+        } else if (metamodel instanceof SchemaSet) {
+          const protocol = new V1_SchemaSet();
+          V1_initPackageableElement(protocol, metamodel);
+          protocol.name = metamodel.name;
+          protocol.package = metamodel.package?.fullPath ?? '';
+          protocol.format = metamodel.format;
+          protocol.schemas = metamodel.schemas;
+          return protocol;
+        }
+        return undefined;
+      },
+    ];
+  }
+  override V1_getExtraConnectionBuilders(): V1_ConnectionBuilder[] {
+    return [
+      (
+        connection: V1_Connection,
+        context: V1_GraphBuilderContext,
+        store?: PackageableElementReference<Store> | undefined,
+      ): Connection | undefined => {
+        if (connection instanceof V1_ExternalFormatConnection) {
+          const Store = !store
+            ? V1_resolveBinding(
+                guaranteeNonNullable(connection.store, 'Binding is missing'),
```

Comment:
Let's think of the user here and write out the full message: `External format connection binding is missing`